lib/Runtime/Types/SimpleTypeHandler.h

```diff
@@ -69,6 +69,8 @@ namespace Js
 
         virtual void SetIsPrototype(DynamicObject* instance) override;
 
+        BOOL IsSimpleTypeHandler() const override { return TRUE; }
```

Comment:
This is a DBG-only method, right? Can we put it under DBG? (Thanks.)

Reply:
Well, you're right that it's only used in an Assert, so I could certainly put it under DBG.  However, I also added the method to DynamicTypeHandler, next to the existing methods IsPathTypeHandler, IsSimpleDictionaryTypeHandler, etc.  Those aren't under DBG.  Would you prefer that I put IsSimpleTypeHandler under DBG or leave it as is for consistency with the others?